storage/txcache/score.go

```diff
@@ -2,10 +2,11 @@ package txcache
 
 import (
 	"math"
```

Comment:
I know I told you to move this in storage rather than core. What was the reason behind this move? The parameters store higher level of info such as gas price and gas limit. Storage is a plugin, **should** not hold elrond business logic. Do we need to reuse this elsewhere?

Reply:
Scoring computation logic was located in `storage/txcache` since its inception. I've just renamed the file in this PR. The cache itself (storage component) depends on these heuristics to guide the eviction process (initial design, not changed in this PR).

Reply:
Now I see all the tx cache components & logic are inside storage package. Something tells me this should be moved maybe in dataRetriever package as it is more strongly coupled to our logic. Along with pruning storer.... Can be left as it is for now.